exporters/FusionRobotExporter/README.md

```diff
@@ -48,6 +48,3 @@ FusionRobotExporter
     ├── AddIn - Manages add-in user interface
     └── Data - Stores robot configuration and writing to BXDA and BXDJ files
 ```
-## Autodesk Fusion 360 API:
-*  [Documentation](https://help.autodesk.com/view/fusion360/ENU/?guid=GUID-7B5A90C8-E94C-48DA-B16B-430729B734DC)
-*  [Object Model](https://help.autodesk.com/cloudhelp/ENU/Fusion-360-API/images/Fusion.pdf)
```

Comment:
Why did you remove the Fusion 360 API documentation?

Reply:
The Fusion 360 API documentation links can go under the `Development Documentation` wiki page.